cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/TwitterSource.java

```diff
@@ -192,6 +192,9 @@ public void onException(Exception e) {
       }
     };
 
+    // Disable chatty logging from twitter4j.
```

Comment:
It's better to do it at the first statement in this method (maybe right after `super.initialize`).